packages/datatrak-web/src/components/Autocomplete.tsx

```diff
@@ -50,9 +50,29 @@ const SelectedOption = styled(OptionWrapper)`
   }
 `;
 
+const Label = styled.span`
+  font-style: ${props => props.theme.typography.fontWeightBold};
+  color: ${props => props.theme.palette.text.primary};
+`;
+
+const Code = styled.span`
+  margin-left: 0.45rem;
+  margin-right: 0.45rem;
+  color: ${props => props.theme.palette.text.secondary};
+  flex: 1;
+`;
+
 const DisplayOption = ({ option, state }) => {
   const { selected } = state;
-  const label = typeof option === 'string' ? option : option.label || option.value;
+  const label =
+    typeof option === 'string' ? (
+      option
+    ) : (
+      <>
+        <Label>{option.label || option.value}</Label>
+        {option.code ? <Code>{'| ' + option.code}</Code> : null}
```

Comment:
This component gets used in several places, so my suggestion here would be to just make the reports page display the new label by either changing the `label` value here:

![Screenshot 2024-07-16 at 4 24 41 PM](https://github.com/user-attachments/assets/4d8ec11d-642e-4f26-8dc8-0d39ef645d79)

or passing through a `renderOption` in this case